locale/de_DE.po

```diff
@@ -103,11 +103,11 @@ msgstr "leerer heap"
 msgid "syntax error in JSON"
 msgstr "Syntaxfehler in JSON"
 
-#: extmod/modure.c:265
+#: extmod/modure.c:161
 msgid "Splitting with sub-captures"
-msgstr "Teilen mit unter-captures"
+msgstr "Splitting with sub-captures"
```

Comment:
Reverted to English?

Reply:
Only when its unclear for me. Better englich than bad translation.